### PR TITLE
IAN QA fixes

### PR DESCRIPTION
**Pull request by pattihis**
### 🎫 Ticket

[TEC-5165]

### 🗒️ Description
Fixing QA issues of In-App-Notifications. Follow up to https://github.com/the-events-calendar/tribe-common/pull/2251

### ✔️ Checklist
- [ ] Ran `npm run changelog` to add changelog file(s). More info [here](https://docs.theeventscalendar.com/developer/git/changelogs/#process)
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

[TEC-5165]: https://stellarwp.atlassian.net/browse/TEC-5165?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/resources/js/ian-client.js [hunk 1]
```diff
@@ -82,15 +89,56 @@
...
 				Ian.sidebar.classList.add("is-hidden");
+				calculateSidebarPosition();
 			}
 		};
 
+		const calculateSidebarPosition = () => {
+			const bar = document.getElementById("wpadminbar");
+			const wrapper = document.querySelector(".wrap .ian-header");
+			if (wrapper) {
+				const rect = wrapper.getBoundingClientRect();
```
Comment (jesseeproductions) [inline src/resources/js/ian-client.js:107]: Is wrapper or settingstabs ever on the same page? 

The way it is setup settingstabs will always set the positions if on the same page, is that correct?

File: src/resources/postcss/tribe-common-admin/settings/_page.pcss [hunk 2]
```diff
@@ -6,66 +6,79 @@
...
-		max-width: 100%;
-		padding: var(--tec-spacer-2) var(--tec-spacer-5);
-		width: auto;
-
-		@media (--viewport-xlarge) {
-			width: 100%;
-		}
-	}
+ .tribe_events_page_tec-events-settings {
+  #wpcontent {
```
Comment (jesseeproductions) [inline src/resources/postcss/tribe-common-admin/settings/_page.pcss:10]: Should this have an indent here?

File: src/resources/js/ian-client.js (hunk 1 above)
Comment (pattihis) [reply src/resources/js/ian-client.js:107]: Hi @jesseeproductions , either will exist, not both. The one is the case in our Settings pages and the other is the case in the native Events Table or Edit Events pages.

File: src/resources/postcss/tribe-common-admin/settings/_page.pcss (hunk 2 above)
Comment (pattihis) [reply src/resources/postcss/tribe-common-admin/settings/_page.pcss]: The linter had formatted it like this, don't know why. Added one indentation, thanks!

File: src/resources/postcss/ian-client.pcss [hunk 3]
```diff
@@ -73,95 +71,107 @@
...
       font-weight: var(--tec-font-weight-bold);
     }
 
     &--description {
-      max-width: 320px;
       margin: 0 auto;
+      max-width: 320px;
 
       p {
+        margin-bottom: 10px;
```
Comment (Camwyn) [inline src/resources/postcss/ian-client.pcss:107]: 8px (--tec-spacer-1) or 12px (--tec-spacer-2)

File: src/resources/postcss/ian-client.pcss [hunk 4]
```diff
@@ -73,95 +71,107 @@
...
+
         font-family: 'Arial', sans-serif;
         font-size: var(--tec-font-size-2);
-        margin-bottom: 10px;
       }
     }
 
     &--button {
+      margin: 16px auto;
```
Comment (Camwyn) [inline src/resources/postcss/ian-client.pcss:115]:
```suggestion
      margin: var(--tec-spacer-3) auto;
```

File: src/resources/postcss/ian-client.pcss [hunk 5]
```diff
@@ -73,95 +71,107 @@
...
       padding: 8px 10px;
```
Comment (Camwyn) [inline src/resources/postcss/ian-client.pcss:116]:
```suggestion
      padding: var(--tec-spacer-1) var(--tec-spacer-2);
```
--tec-spacer-3 is 12px, we don't have a 10px spacer.

File: src/resources/postcss/ian-client.pcss [hunk 6]
```diff
@@ -73,95 +71,107 @@
...
 
-      color: var(--tec-admin-core-blue-50);
+      width: max-content;
 
       border: 1px solid var(--tec-admin-core-blue-50);
       border-radius: 3px;
```
Comment (Camwyn) [inline src/resources/postcss/ian-client.pcss:121]:
```suggestion
      border-radius:var(--tec-border-radius-default);
```
4px

File: src/resources/postcss/ian-client.pcss [hunk 7]
```diff
@@ -73,95 +71,107 @@
...
         background-color: var(--tec-admin-core-blue-50);
+        cursor: pointer;
+
         color: var(--tec-admin-core-grey-0);
       }
     }
 
     &.disable {
-      pointer-events: none;
       opacity: 0.35;
```
Comment (Camwyn) [inline src/resources/postcss/ian-client.pcss:134]:
```suggestion
      opacity: var(--tec-opacity-select-highlighted);
```

File: src/resources/postcss/ian-client.pcss [hunk 8]
```diff
@@ -73,95 +71,107 @@
...
+
     animation: ian-loader 0.5s infinite linear;
   }
   @keyframes ian-loader {
-      100% {background-position: 0 0}
+    100% {background-position: 0 0}
   }
 
   &__notifications {
     padding: 16px;
```
Comment (Camwyn) [inline src/resources/postcss/ian-client.pcss:159]:
```suggestion
    padding: var(--tec-spacer-3);
```

**Review by Camwyn — CHANGES_REQUESTED** [review]
CSS custom props. I stopped flagging them.
Why are we doing CSS display:sticky via javascript?
Some of these classes appear...odd. Left a question on one.

**Review by Camwyn — APPROVED** [review]

**Pull request merged**